.github/workflows/actions.yml

```diff
@@ -5,33 +5,49 @@ on:
   pull_request:
     branches: [master]
 jobs:
-  build:
+  tests:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        ruby: ['2.7', '3.0', '3.1']
-        rails: ['5.0', '5.1', '5.2', '6.0', '6.1', '7.0']
-        task: [test]
+        ruby: ['3.0', '3.1', '3.2']
+        rails: ['6.0', '6.1', '7.0']
         include:
-        # run rubocop against lowest supported ruby
-        - ruby: '2.7'
-          rails: '5.0'
-          task: rubocop
-        exclude:
-        - { ruby: '3.0', rails: '5.0', task: test }
-        - { ruby: '3.0', rails: '5.1', task: test }
-        - { ruby: '3.0', rails: '5.2', task: test }
-        - { ruby: '3.1', rails: '5.0', task: test }
-        - { ruby: '3.1', rails: '5.1', task: test }
-        - { ruby: '3.1', rails: '5.2', task: test }
-    name: ruby${{ matrix.ruby }} rails${{ matrix.rails }} rake ${{ matrix.task }}
+          - {ruby: '2.7', rails: '5.0'}
+          - {ruby: '2.7', rails: '5.1'}
+          - {ruby: '2.7', rails: '5.2'}
+    name: ruby${{ matrix.ruby }} rails${{ matrix.rails }} rake test
     env:
       BUNDLE_GEMFILE: gemfiles/rails${{ matrix.rails }}.gemfile
     steps:
-    - uses: zendesk/checkout@v2
-    - uses: zendesk/setup-ruby@v1
+    - uses: actions/checkout@v3
+    - uses: ruby/setup-ruby@v1
       with:
         ruby-version: ${{ matrix.ruby }}
         bundler-cache: true
-    - run: bundle exec rake ${{ matrix.task }}
+    - run: bundle exec rake test
+
+  linter:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1
+        with:
```

Comment:
should run on 2.7 so it matches the lowest supported ruby version

Reply:
Yep, it will use the version specified in `.ruby-version`.